etc/build/push.sh

```diff
@@ -34,7 +34,7 @@ echo "Version: $VERSION"
 
 ./grab-dependencies.sh
 
-git add .
+git add context/package*.json
```

Comment:
I don't suggest doing this here, but we could consider pulling some of the git version/git tag logic out of this script and into package.json? https://docs.npmjs.com/cli/v7/commands/npm-version details some of the options.

Reply:
I think that'd be a good idea! I made a ticket: https://hms-dbmi.atlassian.net/browse/HMP-410